crates/sargon-uniffi/src/system/sargon_os/sargon_os_transactions.rs

```diff
@@ -1,35 +0,0 @@
-use crate::prelude::*;
-
-// ==================
-// Submit Transaction
-// ==================
-#[uniffi::export]
```

Comment:
This file was no longer used, as its functiones were migrated to different files: `sargon_os_transaction_submit` & `sargon_os_transaction_status`.

It probably slipped after merging main into UniFFI separation branch.